src/ui/components/atoms/checkbox/main.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  stories: ['../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+  webpackFinal: async (config) => {
+    config.module.rules.push({
+      test: /\.(ts|tsx)$/,
+      use: [
+        {
+          loader: require.resolve('babel-loader'),
+          options: {
+            presets: [require.resolve('babel-preset-react-app')],
+          },
+        },
+      ],
+    });
+    config.resolve.extensions.push('.ts', '.tsx');
+    return config;
+  },
+};
```

Comment:
Why do you need this config, this config is made in the storybook please remove it!

Reply:
The setting was to ensure that TypeScript and TSX files are recognized correctly. But I realized that it is already integrated into the storybook, so it is no longer necessary.